lib/manager/batect/types.ts

```diff
@@ -0,0 +1,7 @@
+export interface BatectConfig {
+  containers?: Record<string, BatectContainer>;
+}
+
+interface BatectContainer {
```

Comment:
```suggestion
export interface BatectContainer {
```

should be exported too 😉